packages/features/ee/common/server/LicenseKeyService.ts

```diff
@@ -50,14 +58,13 @@ class LicenseKeyService implements ILicenseKeyService {
       ...options.headers,
       "Content-Type": "application/json",
       nonce: nonce,
-      "x-cal-license-key": this.licenseKey,
+      "x-cal-license-key": licenseKey,
```

Comment:
totally nit: why getting licenseKey as a param while getting signatureToken from this.?